docs/01-app/03-api-reference/04-functions/unstable_forbidden.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: forbidden
+description: API Reference for the forbidden function.
+version: unstable
+related:
+  links:
+    - app/api-reference/file-conventions/forbidden
+---
+
+The `unstable_forbidden` function allows you to render the [`forbidden.js` file](/docs/app/api-reference/file-conventions/forbidden) during authentication.
+
+## Version History
+
+| Version   | Changes                          |
+| --------- | -------------------------------- |
+| `v15.1.0` | `unstable_forbidden` introduced. |
```

Comment:
we can also remove the prefix here. The feature is guarded under the flag `experimental. authInterrupts` in next config